src/main/java/org/folio/rest/support/EffectiveCallNumberComponentsUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.rest.support;
+
+import org.apache.commons.lang.StringUtils;
+import org.folio.rest.jaxrs.model.EffectiveCallNumberComponents;
+import org.folio.rest.jaxrs.model.HoldingsRecord;
+import org.folio.rest.jaxrs.model.Item;
+
+/**
+ * Utility methods for Effective Call Number Components object.
+ */
+public final class EffectiveCallNumberComponentsUtil {
+
+  private EffectiveCallNumberComponentsUtil() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  public static EffectiveCallNumberComponents buildComponents(HoldingsRecord holdings, Item item) {
+    return updateComponents(new EffectiveCallNumberComponents(), holdings, item);
+  }
+
+  public static EffectiveCallNumberComponents updateComponents(EffectiveCallNumberComponents components, HoldingsRecord holdings, Item item) {
```

Comment:
```suggestion
  private static EffectiveCallNumberComponents updateComponents(EffectiveCallNumberComponents components, HoldingsRecord holdings, Item item) {
```

It seems, that this method is used only within the single class.